internal/crossplane/metadata.go

```diff
@@ -90,3 +101,8 @@ func parseBoolLabel(s string, def bool) (bool, error) {
 	}
 	return strconv.ParseBool(s)
 }
+
+func getPlanSize(name string) string {
+	tmp := strings.Split(name, "-")
```

Comment:
Can we have a safe guard here, in case the plan name contains more than one `-`?
I.e. if at some point in the future we have `super-large-premium` 😉 it at least fails with an error...

Reply:
We could also just strip the `-premium` suffix, that's maybe the safer approach...